addons/binding/org.openhab.binding.seneye/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="seneye"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="seneye">
+        <label>Seneye Monitor</label>
+        <description>Seneye Monitor Thing</description>
+
+        <channels>
+            <channel id="channeltemperature" typeId="seneye-temperature-channel"/>
+            <channel id="channelph" typeId="seneye-ph-channel"/>
+            <channel id="channelnh3" typeId="seneye-nh3-channel"/>
+            <channel id="channelnh4" typeId="seneye-nh4-channel"/>
+            <channel id="channelO2" typeId="seneye-O2-channel"/>
+            <channel id="channellux" typeId="seneye-lux-channel"/>
+            <channel id="channelpar" typeId="seneye-par-channel"/>
+            <channel id="channelkelvin" typeId="seneye-Kelvin-channel"/>
+            <channel id="channellastreading" typeId="seneye-lastreading-channel"/>
+            <channel id="channelslideexpires" typeId="seneye-slideexpires-channel"/>
+        </channels>
+        <config-description>
+            <parameter name="aquariumname" type="text" required="true">
+                <label>Aquarium Name</label>
+                <description>The name of your aquarium</description>
+                <advanced>false</advanced>
+                <required>true</required>
+            </parameter>
+
+            <parameter name="username" type="text" required="true">

```

Comment:
thought about this too, add a bridge and it can autodiscover all your aquarium sensors. But I think it's a bit overkill for now. It's always possible to add multiple seneye things (with a different aquariumname parameter)

Reply:
From a modelling perspective the useraccount data belong to a `bridge` thing because this enables you access to the `thing`s behind, i.e. the monitor.

So if you can have multiple aquariums in your account I would definitely vote for having a `bridge` with the account data. The [Tankerkoenig binding](https://github.com/openhab/openhab2-addons/tree/master/addons/binding/org.openhab.binding.tankerkoenig/ESH-INF/thing) is doing that for example.